themes/custom_theme/valencia-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,565 @@
+/* inject:imports */
+
+
+/* endinject */
+
+// Google Fonts
+@import url('https://fonts.googleapis.com/css?family=Roboto+Slab:400,700|Roboto:300,400,500,700');
```

Comment:
We should import the fonts into your theme rather than calling them online if possible.